models.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <title>Drawing with Russians, Models </title>
```

Comment:
Let's call this consistently with what's in index.html. If it's "posting there", this should be Posing too.

```suggestion
    <title>Drawing with Russians, Posing</title>
```